### PR TITLE
Create Meet the Hoodies: Gregor blog post!

### LINKED ISSUE #74: BLOG: Meet the Hoodies – Gregor Martynus
welcome-to-open-source-atlanta

**This issue is reserved for participants of  [Welcome to Open Source, Atlanta](http://www.meetup.com/Jr-Dev-Mentoring/events/228951389/)**.
If it’s still available after April 2nd, it’s all yours :)

---

### :pencil2: The Task

We’ve started a series of brief interviews with some of the Hoodie core contributors. You can see examples of these here:

+ [Alex Feyerke](http://hood.ie/blog/Alex-Feyerke-meet-the-hoodies-1.html)
+ [Charlotte Spencer](http://hood.ie/blog/Charlotte-Spencer-meet-the-hoodies-2.html)
+ [Jenn Turner](http://hood.ie/blog/meet-the-hoodies-jenn-turner.html)  

I’m requesting that someone write up a new interview with Gregor Martynus (@gr2m). To claim this task do the following:

- [x] Claim this issue (@kmcrayton7)
- [x] Review the previous posts above. 
- [x] Find the `Meet the Hoodies` [sample doc in this repo](https://github.com/hoodiehq/editorial/blob/master/documents/meet-the-hoodies.md).
- [x] Contact @gr2m to collaborate on a draft. 
- [x] Submit to @renrutnnej @gr2m or @janl for review (Google docs probably work best for this sort of thing)
- [ ] You are done :clap: comment below that your content is ready for review :tada:

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

### PR DESCRIPTION
**Pull request by jennwrites**
fixes hoodiehq/editorial#74

I stubbed out this blog post but there's still things to be done here for @kmcrayton7 :relaxed:
- [x] Add an intro about interviewing Gregor to the top of the blog post
- [x] Find the link to the sloth image so we can include that
- [x] Address any feedback in the review/PR process 

Please ping me if you need any assistance!

### CODE REVIEW

File: _posts/2016-04-14-meet-the-hoodies-gregor-martynus.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+In an effort not to become attached to a name, the application was initially named CANG (Code Application Next Generation). Which was the worse name we could think of at the time. But once we realized that we had a viable product the name search began. After some time, we were sitting around and Jan pulled the hood of his sweatshirt onto his head and said, “Why not call it hoodie”? Our mission has always been to create something that is accessible and welcoming for anyone, so Hoodie felt right and here we are. 
+
+Currently I manage the code team.
+
+## 3. What’s your favorite use of Hoodie in the wild?
+
+[minutes.io](https://minutes.io/welcome). It has 20,000 active users helping to make sure that Hoodie is based on ‘real’ user experiences. minutes.io is to Hoodie as Basecamp is to Rails.
```
Comment (gr2m) [inline _posts/2016-04-14-meet-the-hoodies-gregor-martynus.md:32]: better say thousands of active users

**Comment by gr2m** [comment]
LGTM now :)

File: _posts/2016-04-14-meet-the-hoodies-gregor-martynus.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## 2. What’s your role/what have you contributed to Hoodie?
+
+My role at Hoodie started before there was a Hoodie, with the [minutes.io](http://hood.ie/blog/minutes-offline-case-study.html) project.
+
+minutes.io is an application I built which allows for meeting minutes to be captured in an easy and effective way. While attending the 2014 CouchConf in Berlin (because minutes.io uses [CouchDB](https://cwiki.apache.org/confluence/display/COUCHDB/Introduction), a NoSQL database) I met [Jan Lehnardt](https://twitter.com/janl). He was interested in further building out the application, so he became my co-founder. I focused on the frontend and Jan handled the back.
+
+In an effort not to become attached to a name, the application was initially named CANG (Code Application Next Generation). Which was the worse name we could think of at the time. But once we realized that we had a viable product the name search began. After some time, we were sitting around and Jan pulled the hood of his sweatshirt onto his head and said, “Why not call it hoodie”? Our mission has always been to create something that is accessible and welcoming for anyone, so Hoodie felt right and here we are.
```
Comment (varjmes) [inline _posts/2016-04-14-meet-the-hoodies-gregor-martynus.md:26]: "was the worse" should be "was the worst"

**Comment by varjmes** [comment]
Just one little comment I noticed.

Also, awesome to finally hear how the name 'Hoodie' came to be!

**Comment by schisepo** [comment]
LGTM - Like Charlotte, I love how the name Hoodie came about. Pretty Cool!

**Comment by jennwrites** [comment]
LGTM, great work Kim and Gregor!

**Comment by jennwrites** [comment]
Let's plan on publishing this Thursday, April 28!

**Comment by schisepo** [comment]
Looks good to me ☺ here's a link to a card with the blog post announcement for Twitter: https://trello.com/c/4RloA9TQ/7-meet-the-hoodies

**Pull request merged**